python/patches/structures.patch

```diff
@@ -0,0 +1,15 @@
+--- env/lib/python3.11/site-packages/instaloader/structures.py	2024-10-12 17:55:45.568444739 -0700
++++ env/lib/python3.11/site-packages/instaloader/structures.py	2024-10-12 17:55:36.588444561 -0700
+@@ -320,10 +320,10 @@
+     def _obtain_metadata(self):
+         if not self._full_metadata_dict:
+             pic_json = self._context.graphql_query(
+-                '2b0673e0dc4580674a88d426fe00ea90',
++                '8845758582119845',
+                 {'shortcode': self.shortcode}
+             )
```

Comment:
You should be able to just use `self._context.doc_id_graphql_query`, rather than forcing `doc_id` for every caller.